commerce_coordinator/apps/commercetools/pipeline.py

```diff
@@ -131,7 +129,7 @@ def run_filter(
 
         try:
             ct_api_client = CommercetoolsAPIClient()
-            order = ct_api_client.get_order_by_id(order_id=order_number)
+            order = ct_api_client.get_order_by_number(order_number=order_number)
```

Comment:
Please see #5 on this [comment](https://github.com/edx/commerce-coordinator/pull/207#discussion_r1582752172)